src/genie/libs/parser/iosxe/show_cable_tdr_interface.py

```diff
@@ -0,0 +1,122 @@
+'''show_cable_tdr_interface.py
+IOSXE parser for the following show command
+	* show cable tdr interface {interface}
+'''
+
+# Python
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Or, ListOf, \
+        Optional, And, Default, Use
+
+# import parser utils
+from genie.libs.parser.utils.common import Common
+
+# =================================================
+# Schema for:
+#   * 'show cable tdr interface {interface}'
+# ==================================================
+class ShowCableTdrIntSchema(MetaParser):
+    schema = {
+        Any(): {
+            ('interface'): str,
+            ('speed'): str,
+            ('date'): str,
+            ('time'): str,
+            'pairs': {
+                Any() : {
+                    ('length'): int,
+                    ('tolerance'): int,
+                    ('remote_pair'): str,
+                    ('status'): str,
```

Comment:
please remove brackets around all these keys in the schema:
`('interface'): str,` -> `'interface': str,`